ovos_tts_plugin_server/__init__.py

```diff
@@ -1,8 +1,14 @@
 import requests
-from ovos_plugin_manager.templates.tts import TTS, TTSValidator
+import random
+from ovos_plugin_manager.templates.tts import TTS, TTSValidator, RemoteTTSException
 
 
 class OVOSServerTTS(TTS):
+    public_servers = [
```

Comment:
do we just want the mimic3 servers here? or should we add piper also now that voices can be changed in config

Reply:
well, im thinking we now want piper servers in here mainly

they just werent there yet when list was created